src/Authenticator/Result.php

```diff
@@ -45,15 +46,29 @@ class Result implements ResultInterface
      */
     protected $_errors = [];
 
+    /**
+     * successful identifier or null
```

Comment:
There seems to be a newline missing

Reply:
i don't understand why.

```
$ composer cs-check
> phpcs --colors -p src/ tests/
............................................................ 60 / 91 (66%)
...............................                              91 / 91 (100%)


Time: 2.68 secs; Memory: 12MB
```

Reply:
Some of those are not properly sniffed yet, missing sniffer rules.
But you can see the other doc lines on how this is handled so far.